src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -86,7 +87,8 @@ class MessagingWidget extends React.Component {
 
     return (
       <div id="msg-widget">
-        <h2>Check Secure Messages</h2>
+        {environment.isProduction() && <h2>Check Secure Messages</h2>}
+        {!environment.isProduction() && <h3>Check secure messages</h3>}
```

Comment:
I'd say unless we're a/b testing this casing and content team doesn't object, we could just roll these casing changes. Would be less post launch clean up.

Reply:
It's also a stylistic change, switching to a smaller heading size since this widget will be nested under a different section.